group.py

```diff
@@ -2,4 +2,29 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {
+    "Jill": {
+        "name": "Jill",
+        "age": 26,
+        "job": "biologist",
```

Comment:
Does not allow people with no job, use list instead (can be empty), such as:
"job" : ["biologist"]

Reply:
thanks m8